src/views/annual-report/2021/l10n.json

```diff
@@ -52,7 +51,8 @@
     "annualReport.2021.reachNewUsers": "New Users",
     "annualReport.2021.reachProjectsCreated": "Projects Created",
     "annualReport.2021.reachProjectCreators": "People Creating Projects",
-    "annualReport.2021.reachScratchAroundTheWorld": "Scratch is used around the world across more than 200 countries and territories",
+    "annualReport.2021.reachScratchAroundTheWorld": "Scratch is used around the world across {bold}",
+    "annualReport.2021.reachScratchAroundTheWorldBold": "more than 200 countries and territories",
```

Comment:
I am curious if this is the best way to do this, or if it will be problematic for the string to be split across two different messages.

Reply:
Actually, we probably want the placeholder name to reflect the content more than the styling (something like `numberOfCountries`.) I'll make that change after I merge this PR.